pkg/operator/operator.go

```diff
@@ -44,10 +47,41 @@ const (
 	cronInterval          = 5  // seconds
 )
 
-var markedWorkflows = strset.New()
+var (
+	awsClient       *aws.Client
+	telemtryClient  *telemetry.Client
+	markedWorkflows = strset.New()
+)
 
 func main() {
-	telemetry.ReportEvent("operator.init")
+	conf := config.NewFromEnv()
+	config.AWS = aws.New(conf.Region, conf.Bucket)
```

Comment:
What do you think about initializing `AWS`, `Telemetry`, and `Kubernetes` in `config.Init()` as well?